scenes/strategic-point-menu.tscn

```diff
@@ -30,7 +30,7 @@ __meta__ = {
 radius = 100
 animation_speed = 0.5
 
-[node name="SlotBottomRight" parent="RadialControl" instance=ExtResource( 1 )]
+[node name="SlotBunker" parent="RadialControl" instance=ExtResource( 1 )]
```

Comment:
Towers may be not in fixed position in every level's strategic menu so i think the names should refer to the position